tests/lit-tests/2844-3.ispc

```diff
@@ -0,0 +1,17 @@
+// Check that we catch incomplete struct types in variable definitions
+
+// RUN: not %{ispc} --target=host --nowrap --nostdlib %s -o %t.o 2>&1 | FileCheck %s
+
+// CHECK: Error: type struct 'Sx' is incomplete and cannot be initialized
+// CHECK: Error: variable 'x' has initializer but incomplete struct type
+int foo() { struct Sx x = { 0 }; }
+// CHECK: Error: type struct 'Sy' is incomplete and cannot be declared
+// CHECK: Error: variable 'y' has incomplete struct type and cannot be defined
```

Comment:
Can be here a case when one error is reported and the other is not? Feels like we only need on error message instead of two. It would be the same behavior as in C++.

Reply:
I have removed the first report.